bugbear.py

```diff
@@ -28,6 +28,14 @@
     ast.GeneratorExp,
 )
 FUNCTION_NODES = (ast.AsyncFunctionDef, ast.FunctionDef, ast.Lambda)
+B908_pytest_functions = {"raises", "warns"}
+B908_unittest_methods = {
+    "assertRaises",
+    "assertRaisesRegex",
+    "assertRaisesRegexp",
+    "assertWarns",
+    "assertWarnsRegex",
```

Comment:
```suggestion
    "assertWarnsRegex",
    "assertWarnsRegexp",
```
Why not

Reply:
There is no such method as I can see https://grep.app/search?q=assertWarnsRegexp
My python3.11 installation has no such method in `unittest`

Reply:
but I can add it just in case